backend/app/api/v1/endpoints/cpt/cptJobs.py

```diff
@@ -102,21 +103,36 @@ async def jobs(
         )
 
     offset, size = normalize_pagination(offset, size)
+    filter_dict = get_dict_from_qs(filter) if filter else {}
+    filter_product = filter_dict.pop("product", None)
+    individual_prod = ["ocp", "telco", "quay"]
+    if filter_product:
+        matched = [p for p in filter_product if p in individual_prod]
+        unmatched = [p for p in filter_product if p not in individual_prod]
+        filter_product = matched + ["hce", "ocm"] if unmatched else matched
+        filter_dict["product"] = unmatched
+
+    prod_list = filter_product if filter_product else list(products.keys())
```

Comment:
I'm losing the logic here -- a comment would help to explain what you're doing. But should this be the `filter_product` (the original input `filter_dict["product"]`), or the *updated* `filter_dict["product"]`?

Reply:
Moved this code to a helper function in `utils.py` with the comments explaining the logic.